extension.js

```diff
@@ -7,6 +7,22 @@ const PanelMenu = imports.ui.panelMenu;
 const Me = ExtensionUtils.getCurrentExtension();
 const workspaceManager = global.workspace_manager;
 
+
+// workspace switch to previous / next
+let scroll_wrap = false;
+function workspace_switch(d){
+  let i = workspaceManager.get_active_workspace_index()
+  let n = workspaceManager.get_n_workspaces()
```

Comment:
I find that single letter variables hurt code readability so how about:

d => index
i => active_index
n => workspace_count
x => target_index

Reply:
I used to think local variables (nearly 10 lines or less) were special cases for short name.

of couse, I'll change it